src/components/NoDataFilter/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import NoData from '../../assets/no-data.jpg';
+import { NoDataContainer, ImageContainer, PlaceHolder } from './style';
+
+function NoDataFilter({ text }) {
```

Comment:
Shouldn't it be like EmptyData or something more meaningful beacause what i thing after seeing this is not telling me what this component is actually doing.